gui/list.go

```diff
@@ -98,17 +97,17 @@ func (m *model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			m.metadataLoopChan <- struct{}{} // Stop the existing loop
 			go UpdateMetadataLoop(m, 0)      // Start the new one
 		case "+", "-":
-			if msg.String() == "+" && m.volume < 100 {
+			if msg.String() == "+" && m.volume < 150 {
```

Comment:
Setting max volume to 100 was a deliberate choice to avoid distortions

Reply:
done